recipes/tcpcat/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.15)
+project(PackageTest CXX)
+
+add_executable(test_package test_package.cpp)
+
+find_package(tcpcat CONFIG REQUIRED)
+target_link_libraries(test_package tcpcat::tcpcat)
```

Comment:
In https://github.com/ydcpp/tcpcat/blob/main/test/CMakeLists.txt#L26 you use `tcpcat` as the target name without the namespaces. What's your expected name?
If you prefer to have it as shown in the tests, we can acomplish that by adding a `self.cpp_info.set_property("cmake_target_name", "tcpcat")` line in the conanfile's `package_info()` method

Reply:
thank you for the suggestion, yes we can add this to conanfile's `package_info()` method